### PR TITLE
PHP: Use phpcs for PhpSdkDev Linting

### PR DESCRIPTION
**Pull request by charjr**
Add PHP Code Sniffer
  - Configured to lint against standards: PSR1 and PSR12

Can be called through the PhpSdkDev.Lint

Added to to Dev module in the same way as Python's Linting works.

### CODE REVIEW

File: sdk/php/composer.json
Comment (charjr) [inline sdk/php/composer.json:32]: `symfony/console` has been removed from `require-dev` because it is already in the `require` section, making this line unnecessary as the package is already required.

File: sdk/php/src/Connection/version.php
Comment (helderco) [inline sdk/php/src/Connection/version.php:3]: This might produce a linting error when the version is bumped via the release process. Should update the code that generates this in `/dev/sdk_php.go` so that it doesn't produce a linting error.

**Review by helderco — COMMENTED** [review]
Looks good! 

If you make a follow up to delegate codegen to `/sdk/php/dev`, then we don't need the duplicated base in `/dev/sdk_php.go` anymore. 🙂

File: dev/sdk_php.go
Comment (charjr) [reply dev/sdk_php.go]: I've now made this change. 

For future reference, may you please explain why this change was suggested?

Comment (helderco) [reply dev/sdk_php.go]: Not important. It's just more conventional to put the usual `if err != nil { return nil }` error handling just below the statement that returns the error, especially if it's a one liner.

File: sdk/php/src/Connection/version.php
Comment (helderco) [reply sdk/php/src/Connection/version.php:3]: FTR, it just means you need to adjust the following to look like that (add two `\n` after `<?php`):

https://github.com/dagger/dagger/blob/0f8b9d58535fcf65d560448ce4e31dfebecdfdac/dev/sdk_php.go#L102-L103

File: dev/sdk_php.go
Comment (helderco) [inline dev/sdk_php.go:120]: You removed a space there. And should also probably add a newline, according to the linter:
```
FILE: /src/sdk/php/src/Connection/version.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 3 | ERROR | [x] Expected 1 newline at end of file; 0 found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```
```suggestion
		"<?php\n\n/* Code generated by dagger. DO NOT EDIT. */\nreturn '%s';\n",
```

File: sdk/php/dev/src/PhpSdkDev.php
Comment (helderco) [inline sdk/php/dev/src/PhpSdkDev.php:34]: You need a "format" function as well. It'll make it easier for other maintainers that get PHP linting errors, to run the formatter in order to solve them.

You can return a Directory with the diff of the changes, like this:

https://github.com/dagger/dagger/blob/b0d27e15843341643954a4e9ef0841cd785b78b3/sdk/python/dev/src/main/__init__.py#L186-L187

**Review by helderco — APPROVED** [review]
It'd be great if you can include the formatter, but don't want to block on it so you can do it in a follow up.

File: sdk/php/dev/src/PhpSdkDev.php
Comment (charjr) [reply sdk/php/dev/src/PhpSdkDev.php:34]: I've added #8124 to ensure I get that done in a follow up.

If we can get this merged separately the `format()` should be a fairly small PR.

**Comment by charjr** [comment]
@helderco I've created two issues for the suggested follow ups: #8124 and #8125.

I will do those separately so we can keep the PRs small and easier for review.

**Pull request merged**
